src/buttons/ChatboxButton/ChatboxButton.tsx

```diff
@@ -1,23 +1,69 @@
-import { IconSizeVariant } from '@/types';
+import { ActionButton, ActionButtonVariant, IconSizeVariant } from '@/types';
 
-import ForumIcon from '@mui/icons-material/Forum';
-import { IconButton, Tooltip } from '@mui/material';
+import { Chat } from '@mui/icons-material';
+import SpeakerNotesOffIcon from '@mui/icons-material/SpeakerNotesOff';
+import { IconButton, SvgIconOwnProps, SxProps, Tooltip } from '@mui/material';
+
+import MenuItemButton from '../MenuItemButton';
 
 export type Props = {
   id?: string;
   onClick?: () => void;
   size?: IconSizeVariant;
   tooltip?: string;
+  menuItemClassName?: string;
+  showChat: boolean;
+  showChatText?: string;
+  hideChatText?: string;
+  type?: ActionButtonVariant;
+  color?: SvgIconOwnProps['color'];
```

Comment:
In the Item badge you do not use this `color` prop but you set in with the `sx` why is that ?

Reply:
okay 👌